tools/install_demo_configuration.sh

```diff
@@ -165,16 +166,16 @@ fi
 ES_CONF_DIR=$(dirname "${ES_CONF_FILE}")
 ES_CONF_DIR=`cd "$ES_CONF_DIR" ; pwd`
 
-if [ ! -d "$ES_PLUGINS_DIR/opendistro_security" ]; then
+if [ ! -d "$ES_PLUGINS_DIR/$PLUGIN_NAME" ]; then
   echo "Open Distro Security plugin not installed. Quit."
   exit -1
 fi
 
 ES_VERSION=("$ES_LIB_PATH/elasticsearch-*.jar")
 ES_VERSION=$(echo $ES_VERSION | sed 's/.*elasticsearch-\(.*\)\.jar/\1/')
 
-SECURITY_VERSION=("$ES_PLUGINS_DIR/opendistro_security/opendistro_security-*.jar")
-SECURITY_VERSION=$(echo $SECURITY_VERSION | sed 's/.*opendistro_security-\(.*\)\.jar/\1/')
+SECURITY_VERSION=("$ES_PLUGINS_DIR/$PLUGIN_NAME/opendistro-security-*.jar")
+SECURITY_VERSION=$(echo $SECURITY_VERSION | sed 's/.*opendistro-security-\(.*\)\.jar/\1/')
```

Comment:
Why leave the jar file name hardcoded instead of using the `$PLUGIN_NAME` variable you created earlier?

Reply:
This is actually not plugin name. This is the artifact name.